packages/postgresql-debversion-evr/postgresql-debversion-evr.spec

```diff
@@ -0,0 +1,63 @@
+%if 0%{?rhel} <= 7
+%global scl rh-postgresql12
+%endif
+
+%{?scl:%scl_package postgresql-debversion-evr}
+%{!?scl:%global pkg_name %{name}}
+
+Name:     %{?scl_prefix}postgresql-debversion-evr
+Version:  0.0.1
+Release:  1%{?dist}
+Summary:  Debian version number compare function for PostgreSQL
+
+Group:    Applications/System
+License:  GPLv3
+URL:      https://github.com/ATIX-AG/postgresql-debversion-evr
+Source:   https://codeload.github.com/ATIX-AG/postgresql-debversion-evr/tar.gz/%{version}#/postgresql-debversion-evr-%{version}.tar.gz
+
+Requires: %{?scl_prefix}postgresql-server
+%{?scl:Requires: %{?scl_prefix}runtime}
+
+%{?scl:BuildRequires: %{?scl_prefix}runtime}
+BuildRequires: %{?scl_prefix}postgresql-devel
+
+%if 0%{?rhel} > 7
+BuildRequires: postgresql-server-devel
+%endif
+
+ExclusiveArch: x86_64
```

Comment:
What makes this limited to this arch? Since it's pure SQL, isn't it noarch in the end?

Reply:
Shamelessly copied form `postgresql-evr`'s SPEC :innocent: 
https://github.com/theforeman/pulpcore-packaging/blob/d081221df48e7c539d9ba9b4d742f237e40ae949/packages/postgresql-evr/postgresql-evr.spec#L28
Though I guess you are right about this.